src/main/java/io/unlogged/Runtime.java

```diff
@@ -69,8 +71,17 @@ private Runtime(String args) {
                 }
             }
 
+			String hostname = null;
+			InetAddress inetAddress;
+			try {
+				inetAddress = InetAddress.getLocalHost();
+				hostname = inetAddress.getHostName();
+			} catch (UnknownHostException e) {
+				e.printStackTrace();
```

Comment:
send error to logger

Reply:
done [here](https://github.com/unloggedio/unlogged-sdk/pull/49/commits/2271796e88fe8e2049b3424ff4c48f8b210faf04#diff-604b05d77d6594ce6d9f0c9a2e17c0eddef5eeef74ebefa4c130555c84362f4fR97-R100).